Open_Science_New

```diff
@@ -261,21 +246,21 @@
 	<section>
 	  <section>
 	    <h2>Standardization</h2>
-	    <img src="images/INCF_2019.png">
+	   <a href="https://www.incf.org"><img src="images/INCF_2019.png"></a>
 	  </section>
 	  
 	  <!-- Standarization frameworks-->
 	  <section data-transition="fade">
 	    <table class="reveal">
 	      <tr>
 		<td width="25%" style="vertical-align: middle">
-		  <img src="images/Nidm.png">
-		  <img src="images/BIDS.png">
+		  <a href="http://nidm.nidash.org/specs/nidm-overview.html"><img src="images/Nidm.png"></a>
+		  <a href="https://bids.neuroimaging.io"><img src="images/BIDS.png"></a>
```

Comment:
Add the brand new BIDS logo where you can, it was just released in June. 
* black version is found on [their wikipedia page](https://en.wikipedia.org/wiki/Brain_Imaging_Data_Structure)
* white version should be downloadable from https://bids.neuroimaging.io/

Reply:
thanks!